sale_block_no_stock/views/res_config_settings_views.xml

```diff
@@ -0,0 +1,52 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+    <record id="res_config_settings_view_form" model="ir.ui.view">
+        <field name="name">res.config.settings.view.form.inherit.sale.block</field>
+        <field name="model">res.config.settings</field>
+        <field name="inherit_id" ref="sale.res_config_settings_view_form" />
+        <field name="arch" type="xml">
+            <xpath expr="//div[@id='no_edit_order']" position="after">
+                <div class="col-12 col-lg-6 o_setting_box">
+					<div class="o_setting_right_pane">
+						<label for="sale_line_field_block" string="Blocking sales due to lack of stock" />
+                        <span
+                            class="fa fa-lg fa-building-o"
+                            title="Values set here are company-specific."
+                            groups="base.group_multi_company"
+                        />
+						<div class="text-muted">
+							Avoid selling more product than you have in stock or planned
+						</div>
+						<div class="content-group">
+							<div class="mt-2">
+                                <label
+                                    for="sale_line_field_block"
+                                    string="Field to compare against the quantity demanded"
+                                    class="w-100"
+                                />
+                                <field
+                                    name="sale_line_field_block"
+                                    options="{'no_create': True, 'no_open': True}"
+                                    class="w-75"
+                                />
+                            </div>
+                            <div class="mt-1">
+                                <label
+                                    for="sale_line_block_allowed_groups"
+                                    string="Groups allowed to bypass the block"
+                                    class="w-100"
+                                />
+                                <field
+                                    name="sale_line_block_allowed_groups"
+                                    widget="many2many_tags"
+                                    options="{'no_create': True, 'no_open': True}"
+                                    class="w-100"
+                                />
+							</div>
+						</div>
+					</div>
+				</div>
+            </xpath>
```

Comment:
Here too

Reply:
It's normal because I'm only putting thinks onto right_pane